content/realtime/channel-params.textile

```diff
@@ -57,7 +57,7 @@ h3. SSE example
 For example, to specify the @rewind@ channel param with the value @"1"@ using a querystring parameter, where it will apply to all channels:
 
 ```[javascript]
-var querystring = 'v=1.1&channels={{RANDOM_CHANNEL_NAME}}&rewind=1&key={{API_KEY}}';
+var querystring = 'v=1.2&channels={{RANDOM_CHANNEL_NAME}}&rewind=1&key={{API_KEY}}';
```

Comment:
Does SSE suport 1.2?

Reply:
Not yet, as I mentioned in [my review of this pr](https://github.com/ably/docs/pull/843#pullrequestreview-384134783)